src/components/home/index.js

```diff
@@ -4,7 +4,7 @@ import NewMembers from '@components/new-members';
 import UserProfile from '@components/user-profile';
 import MemberRoleUpdate from '@components/member-role-update';
 import Designers from '@components/designers';
-import SearchBox from '@components/UI/search-box/index';
```

Comment:
Is there any reason we are not modifying the previous search box?

Reply:
Will delete the previous file .